qemu/tests/cfg/blockdev_mirror_install.cfg

```diff
@@ -0,0 +1,74 @@
+# Storage backends:
+#   filesystem, iscsi_direct, ceph, nbd, gluster_direct
+# The following testing scenario is covered:
+#   Block mirror with vm installation test
+#     The mirror image is a local filesystem image
+
+- blockdev_mirror_install:
+    only Linux
+    no RHEL.5 RHEL.6 RHEL.7 RHEL.8.1
+    only filesystem iscsi_direct ceph nbd gluster_direct
+    type = blockdev_mirror_install
+    qemu_force_use_drive_expression = no
+    need_install = yes
+    start_vm = no
+    kill_vm = yes
+    virt_test_type = qemu
+    source_images = image1
+    target_images = mirror1
+    force_create_image_image1 = yes
+    medium = cdrom
+    installation = cdrom
+    kernel = vmlinuz
+    initrd = initrd.img
+    image_aio = threads
+    unattended_delivery_method = cdrom
+    cdroms += " unattended"
+    index_enable = no
+    shutdown_cleanly = no
+    backup_options_image1 = sync
+    sync = full
+    storage_pools = default
+    storage_pool = default
+
+    image_format_mirror1 = qcow2
+    image_name_mirror1 = mirror1
+
+    tag_for_install_start = "Starting Login Service"
+    rebase_mode = unsafe
+
+    # Always access cd1 and unattended as local storage
+    enable_nbd_cd1 = no
+    enable_nbd_unattended = no
+    enable_gluster_cd1 = no
+    enable_gluster_unattended = no
+    enable_ceph_cd1 = no
+    enable_ceph_unattended = no
+    enable_iscsi_cd1 = no
+    enable_iscsi_unattended = no
+
+    # Install VM on a new image, which can avoid the
+    # original system image crash when hitting a failure
+    image_name_image1 = "images/system_image"
+    gluster_direct:
+        timeout_for_install_done = 3600
+    iscsi_direct:
+        lun_image1 = 1
+        image_size_mirror1 = 21G
+    nbd:
+        force_create_image_image1 = no
+        qcow2:
+            nbd_port_image1 = 10820
+        raw:
+            nbd_port_image1 = 10819
+        luks:
+            nbd_port_image1 = 10821
+            image_secret_image1 = redhat
+
+    # For local mirror images
+    storage_type_default = directory
+    enable_iscsi_mirror1 = no
+    enable_ceph_mirror1 = no
+    enable_gluster_mirror1 = no
+    enable_nbd_mirror1 = no
+    image_raw_device_mirror1 = no
```

Comment:
You could try to add the following cfg instead of avocad-vt patch(https://github.com/avocado-framework/avocado-vt/pull/2761).
```
    variants:
        # Below variants is to share configurations related to installation defined.
        - @with_installation:
            variants:
                - @extra_cdrom_ks:
```
Please refer to : https://github.com/autotest/tp-qemu/pull/2327

Reply:
Cool, let me have a try

Reply:
change done, testing now